generate_engagement_analysis_files.py

```diff
@@ -103,3 +103,54 @@
 
         for row in opted_in_uids_per_show.items():
             writer.writerow(row)
+
+    log.info(f'Computing new and repeat participation for TV episodes ...')
+    # Computes the number of new and repeat consented individuals who participated in the TV_radio shows vis-à-vis
```

Comment:
Is "TV_radio" shows that went out over tv and radio (vs. just the radio)?

Reply:
Yes but vs just the radio shows that aired before the first tv_radio show(mag 23)